utils/frame/benchmarking-cli/src/overhead/weights.hbs

```diff
@@ -23,6 +24,7 @@
 //! WARMUPS: `{{params.bench.warmup}}`, REPEAT: `{{params.bench.repeat}}`
 //! WEIGHT-PATH: `{{params.weight.weight_path}}`
 //! WEIGHT-METRIC: `{{params.weight.weight_metric}}`, WEIGHT-MUL: `{{params.weight.weight_mul}}`, WEIGHT-ADD: `{{params.weight.weight_add}}`
+//! WASM-EXECUTION-METHOD: `{{execution_strategy}}`
```

Comment:
I just noticed that this always prints something, even when `--execution native` is used.  
I'm not sure if its possible to print the executor name so `Native` or `WASM` instead of the WASM strategy, but would be nice.